src/main/resources/db/migration/common/V1_110__insert_automated_task_data.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO darts.automated_task (aut_id,task_name,task_description,cron_expression,cron_editable)
+VALUES (1,'ProcessDailyList','Process the latest daily list for each courthouse','0 2 * * * *',true);
```

Comment:
If we have this in our main migration SQL then I suppose all test environments will have this scheduled task enabled. Maybe check with test team whether this is desirable, as it may be better for them to only add these tasks to the DB when needed.

Reply:
I have raised the question about adding a new field enabled which would mean this could be different for each environment accordingly.